docs/mail.md

```diff
@@ -129,4 +129,11 @@ when is set to 1, that field \`money\` stores gold for COD.
 
 ### auctionId
 
-`field-no-description|14`
+Only if [stationery][3] is 62.
+
+Lot id from AuctionHouse. Can be negative vector in case of delayed mail with money sended by Auction to Lot-owner.
```

Comment:
you mean "loot id" ?

Reply:
no, ID of auction item (Lot) 
first field in auctionhouse table